people/will/homework/hw_1.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Testing Page for Scripts</title>
+</head>
+<body>
+
+<!-- Testing page to try out if and else statements-->
+
+<script type="text/javascript">
+	
+	function powerCheck() {
+	if (document.power1.text1.value > 9000) {
```

Comment:
Nice work! I think this would be better done by caching the element and then referencing it's values as needed. Give the input an ID and then try something like this:

```
var powerLevel = document.getElementById('power');

if (powerLevel.value > 9000) {
  // do something
}
```

You can then try:
`console.log(powerLevel);`

To see what other parameters you can mess with 😄